ovn-tester/ovn_workload.py

```diff
@@ -462,65 +535,70 @@ def default_deny(self):
             'allow')
 
     @ovn_stats.timeit
-    def allow_within_namespace(self):
+    def allow_within_namespace(self, family=4):
```

Comment:
I think we don't need the default `family=4`.  Let's just pass it explicitly.

Reply:
@dceara ack, I will fix it